src/ReactiveUI.Wpf/Themes/Generic.xaml

```diff
@@ -458,7 +458,7 @@
                                     <Storyboard>
                                         <DoubleAnimation 
                                             BeginTime="00:00:00" Duration="00:00:00.2"
-                                            Storyboard.TargetName="PART_PreviousContentPresentationSite" 
```

Comment:
There are frameworks out there using these storyboard names at the moment so probably best you don't rename them for the moment.

Reply:
@glennawatson Type changes anyway ContentPresenter -> Image